docs/source/code_overview/codes_supporting_lasy.rst

```diff
@@ -1,4 +1,17 @@
-Codes Supporting LASY Input Data
-================================
+Codes Supporting LASY Data
+==========================
 
-A list of codes which support LASY Data as input.
+A list of codes which support LASY Data as input:
+
+* Many Coming Soon!
+
+A list of codes currently in the process of adding support:
+
+* `FBPIC <https://github.com/fbpic/fbpic>`_
+* `Wake-T <https://github.com/AngelFP/Wake-T>`_
+* `WarpX <https://github.com/ECP-WarpX/WarpX>`_
```

Comment:
Should we add HiPACE++ (support for `lasy` is complete in this case, in my understanding)

Reply:
Yup, I think we should, thanks! Further tests to be done soon.